src/Core/Domain/Order/QueryResult/OrderCustomerForViewing.php

```diff
@@ -116,7 +128,9 @@ public function __construct(
         ?string $privateNote,
         bool $isGuest,
         string $ape = '',
-        string $siret = ''
+        string $siret = '',
+        int $idDefaultGroup,
+        string $defaultGroupName
```

Comment:
You can't add parameters without default value, after parameter with default value.

Reply:
It's a BC Break as you change the signature.

Reply:
@Progi1984 Should I just add empty default values and it will be fine?

Reply:
Also, keep in mind that adding non-optional parameters after optional parameters is deprecated since PHP 8 ;)

Reply:
@atomiix @Progi1984 Can you help me with completing this PR? What should I do? :-)

Reply:
@Hlavtox I think what should be done is
```
public function __construct(
        int $id,
        string $firstName,
        string $lastName,
        string $gender,
        string $email,
        DateTimeImmutable $accountRegistrationDate,
        string $totalSpentSinceRegistration,
        int $validOrdersPlaced,
        ?string $privateNote,
        bool $isGuest,
        string $ape = '',
        string $siret = '',
        int $idDefaultGroup = null,
        string $defaultGroupName = null
    )
```

Then in your code, you must handle the case `if (null === $idDefaultGroup)`, what should happen in this case 😉